### PR TITLE
Cosmetics docker-compose & native setup

### PR DESCRIPTION
**Pull request by slawosz**

### CODE REVIEW

File: cosmetics-web/.gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.env.development
+.env.test
```
Comment (scruti) [inline cosmetics-web/.gitignore]: :clap:

Comment (scruti) [inline cosmetics-web/.gitignore]: :clap:

File: README.md [hunk 2]
```diff
@@ -7,19 +7,38 @@
...
 
-This project includes [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules), so when running `git clone` you need to add the `--recurse-submodules` option.
-If you have already pulled, you can run `git submodule init` and then `git submodule update --recursive` instead.
-You might also need to run `git submodule update --recursive` if the submodule is updated because of a pull.
+### Quick steps
+
+Add required hosts in `/etc/hosts`:
+
+```
+127.0.0.1       search_cosmetics
```
Comment (slorek) [inline README.md]: This is why we shouldn't rely on `localhost` in https://github.com/UKGovernmentBEIS/beis-opss/pull/1662 /cc @scruti

Comment (scruti) [reply README.md]: Noted!

**Pull request merged**